tomls/markets/perps/btc.toml

```diff
@@ -49,7 +49,7 @@ func = "addSettlementStrategy"
 args = [
     "<%= settings.btcPerpsMarketId %>",
     # strategyType = 0 (pyth)
-    { strategyType = "0", settlementDelay = "<%= settings.bigCapSettlementDelay %>", settlementWindowDuration = "<%= settings.bigCapSettlementWindowDuration %>", priceWindowDuration = "<%= settings.bigCapPriceWindowDuration %>", priceVerificationContract = "<%= settings.pythPriceVerificationAddress %>", feedId = "<%= settings.pythBtcFeedId %>", url = "<%= settings.pythFeedUrl %>", settlementReward = "<%= settings.settlementReward %>", priceDeviationTolerance = "<%= parseEther('1') %>", disabled = false }
+    { strategyType = "0", settlementDelay = "<%= settings.bigCapSettlementDelay %>", settlementWindowDuration = "<%= settings.bigCapSettlementWindowDuration %>", priceWindowDuration = "<%= settings.bigCapPriceWindowDuration %>", priceVerificationContract = "<%= settings.pythPriceVerificationAddress %>", feedId = "<%= settings.pythBtcFeedId %>", url = "<%= settings.pythFeedUrl %>", settlementReward = "<%= parseEther(settings.settlementReward) %>", priceDeviationTolerance = "<%= parseEther('1') %>", disabled = false }
```

Comment:
I suggest not to wrap settings into `parseEther` to avoid confusion. And instead use `parseEther` in settings itself `defaultValue = "<%= parseEthers() %>"`
This change introduces variability in how settings are used as some of them are already in wei and some are plain numbers.

Reply:
I have somewhat related idea (not yet formed) on how to use something like `zod` schema to validate values, so they do come in at least expected ranges.
This is not part of generic cannon setup but perhaps can be implemented as a synthetix-specific plugin? So we declare, let's say, that settlementReward is expected to be in  wei, and if we detect values that are not in a _reasonable_ wei range - validation fails.
This schema can potentially be half-auto extracted from solidity. And/or at least be collocated with contract itself.
Would appreciate any feedback on the idea so we can at least avoid wildly incorrect valued sent to our contracts by mistake